src/services/ModelTasks.ts

```diff
@@ -68,4 +67,44 @@ export class GetOpenedFilesTask extends Task<ModelData, ModelTaskMetadata> {
         return new ModelTaskMetadata([this.mComponent, ModelComponent.IO], [ModelComponent.IO]);
     }
 
+}
+
+/*
+ * Load a certain ModelComponent's data from the model
+ */
+export class LoadComponent extends Task<ModelData, ModelTaskMetadata> {
```

Comment:
If I understand correctly, the `LoadComponent` task just visualizes the graph, right? So why not rename it to `VisualizeGraphTask` or something? Also, maybe give this thing a more descriptive documentation comment. Virtually all tasks load a certain `ModelComponent`'s data from the model.